pkg/controller/tas/node_failure_controller.go

```diff
@@ -176,6 +176,22 @@ func (r *nodeFailureReconciler) getWorkloadsOnNode(ctx context.Context, nodeName
 	return workloadsToProcess, nil
 }
 
+// evictWorkload idempotently evicts the workload when the node has failed.
+// It returns whether the node was evicted, and whether an error was encountered.
+func (r *nodeFailureReconciler) evictWorkload(ctx context.Context, wl *kueue.Workload, nodeName string, wlKey types.NamespacedName) (bool, error) {
+	if failedNode, ok := getAnnotations(wl)[kueuealpha.NodeToReplaceAnnotation]; ok && failedNode != nodeName && !workload.IsEvicted(wl) {
+		r.log.V(3).Info("Evicting workload due to multiple node failures", "workload", wlKey)
```

Comment:
As we are on that, can we log both the names for debuggability? ("Evicting workload due to multiple node failures, including %s, %s"

Reply:
Is a new key "nodeName" in the log fine?

Reply:
Maybe "nodeName", and "failedNode" to include both?

Reply:
Will these ever be different?